packages/diffutils.rb

```diff
@@ -3,21 +3,13 @@
 class Diffutils < Package
   description 'GNU Diffutils is a package of several programs related to finding differences between files.'
   homepage 'http://www.gnu.org/software/diffutils/'
```

Comment:
This is redirecting to `https://www.gnu.org/software/diffutils/` so might as well change this also.